docs/docs/reach-router-and-gatsby.md

```diff
@@ -2,7 +2,22 @@
 title: "@reach/router and Gatsby"
 ---
 
-This is a stub. Help our community expand it.
+This guide gives a peek under the hood of using `@reach/router` with Gatsby.
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Why do we use @react/router?
+
+The main reasons Gatsby uses `@reach/router` are:
+
+1. Preloading. You can read more about Preloading In the docs for the [Gastby Link api](https://www.gatsbyjs.org/docs/gatsby-link/).
```

Comment:
Maybe something like this? 

```md
1. Preloading. You can read more about preloading in the docs for the [Gatsby Link API](https://www.gatsbyjs.org/docs/gatsby-link/). 
```

I also think it'd be nice to add a bit more context here as well.

Reply:
Great grammar check, thank you.